src/gui/properties/propertieswidget.cpp

```diff
@@ -142,7 +142,7 @@ PropertiesWidget::PropertiesWidget(QWidget *parent)
     m_peerList = new PeerListWidget(this);
     m_ui->vBoxLayoutPeerPage->addWidget(m_peerList);
     // Tab bar
-    m_tabBar = new PropTabBar();
+    m_tabBar = new PropTabBar(nullptr);
```

Comment:
Why this one has no parent?

Reply:
There is no correct parent to put in. using `this` will give an warning a runtime: "QLayout: Attempting to add QLayout "" to PropertiesWidget "PropertiesWidget", which already has a layout".